### PR TITLE
Switch to drupal_ti

### PR DESCRIPTION
**Pull request by fabianx-ai**

### CODE REVIEW

**Comment by fabianx-ai** [comment]
Tests fail, but only because pathauto simpletest has an exception, which is counted as fail.

drupal_ti itself works properly now :).

Note: phpunit-core won't do anything by itself when not used in combination with 'simpletest'.

This will be solved upstream in https://github.com/LionsAd/drupal_ti/issues/17

**Comment by Berdir** [comment]
Thanks. The remaining test fail is actually caused by token.module and I've fixed it in 8.x-1.x: https://github.com/md-systems/pathauto/commit/ae3cefc5c15118dfb40de48b800230e6e22ae4d8

Can you switch to that too then I can merge this.

File: .travis-before-script.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+# Add needed dependencies.
+cd "$DRUPAL_TI_DRUPAL_DIR"
+
+# These variables come from environments/drupal-*.sh
+mkdir -p "$DRUPAL_TI_MODULES_PATH"
+cd "$DRUPAL_TI_MODULES_PATH"
+
+# Download token 8.x-1.x
+git clone --depth 1 --branch 8.x-1.x http://git.drupal.org/project/token.git
```
Comment (fabianx-ai) [inline .travis-before-script.sh]: @Berdir This already uses the 8.x-1.x version of token. So this sohuld be good to go, I am sending it for a re-test though.

**Comment by fabianx-ai** [comment]
OH, @Berdir I should switch to the md-systems fork, now I got it ... Fixing.

**Comment by fabianx-ai** [comment]
And green! :)

**Comment by jibran** [comment]
@LionsAd what is the difference between https://github.com/drupal-media/entity_embed/pull/147 and this approach?

**Comment by Berdir** [comment]
Thanks, merged.

**Comment by fabianx-ai** [comment]
@jibran The approach is basically the same, but instead of adding phpunit testing manually, a new runner is created and 'hook'ed into the 'before' directory.

Such that phpunit tests can be run in a Drupal 8 Core context.

**Pull request merged**